components/onboarding_tasks/onboarding_tasks_manager.tsx

```diff
@@ -77,17 +82,34 @@ const taskLabels = {
             defaultMessage='Visit the System Console to configure your workspace'
         />
     ),
+    [OnboardingTasksName.START_TRIAL]: (
+        <FormattedMessage
+            id='onboarding_checklist.task_start_trial'
+            defaultMessage='Learn more about Enterprise-level high-security features'
```

Comment:
It looks like the message id's for the tasks were update to the format `onboardingTask.checklist.task_...`